src/components/seo.js

```diff
@@ -52,6 +52,14 @@ function SEO({ description, lang, meta, title }) {
           property: `og:type`,
           content: `website`,
         },
+        {
+          property: `og:image`,
+          content: "https://i.postimg.cc/Kc3m1JfL/logo-ziggurat.png"
```

Comment:
Why are we using an external link?

Reply:
Tried using the `raw.githubusercontent` link. It errors out on the Facebook debugger. Didn't have a permanent link for the Ziggurat logo anywhere so made one.